templates/provider/kcm/values.yaml

```diff
@@ -33,13 +33,39 @@ controller:
+
+  local: # @schema type: object; description: Local telemetry collection settings, applicable only if .telemetry.mode is set to local; additionalProperties: false
     baseDir: "/var/lib/telemetry" # @schema type: string; title: Base directory where to put local telemetry data; description: Base dir for the local telemetry data, taken into account only if telemetry.mode is set to 'local'; required: true
 
+    volume: # @schema type: object; description: Volumes and claims configuration depending on the source of the storage; additionalProperties: false
+      source: hostPath # @schema type: string; enum: [pvc, existing, hostPath]; description: Defines the source of the storage. [pvc] - chart creates a PVC only (dynamic provisioning via StorageClass). [existing] - chart uses an existing PVC. [hostPath] - chart creates a hostPath source typed PV and matching PVC (default)
+
+      pvc: # @schema type: object; description: PVC settings if source of the storage is set to pvc (dynamic provisioning); additionalProperties: false
+        storageClassName: "" # @schema type: string; description: Name of the StorageClass, empty defines to use default StorageClass
+        accessModes: ["ReadWriteOnce"] # @schema item: string; itemEnum: [ReadWriteOnce, ReadWriteOncePod, ReadOnlyMany, ReadWriteMany]; description: Types of access modes required, RWOP is not supported with the default Deployment settings
+        volumeMode: Filesystem # @schema type: string; enum: [Filesystem, Block]; description: Defines what type of volume is required by the claim
+        size: 200Mi # @schema type: string; description: Defines the requested size required by the claim
+        annotations: {} # @schema description: Annotations (map) to be set; type: object; additionalProperties: true
+
+      existingClaim: "" # @schema description: Name of the claim to be used, valid if source of the storage is to existing; type: string
+
+      hostPath: # @schema type: object; description: Settings of the hostPath PV and PVC source if source of the storage is set to hostPath; additionalProperties: false
+        name: "" # @schema type: string; description: Name of the PV to be created, defaults to <fullname>-telemetry-data
+        reclaimPolicy: Retain # @schema type: string; enum: [Retain, Delete]; description: Policy for end-of-life maintenance of the volume
+        storageClassName: "manual" # @schema type: string; description: Name of the StorageClass to match PV and PVC
+        accessModes: ["ReadWriteOnce"] # @schema item: string; itemEnum: [ReadWriteOnce, ReadWriteOncePod, ReadOnlyMany, ReadWriteMany]; description: Ways the volume can be mounted, RWOP is not supported with the default Deployment settings
+        volumeMode: Filesystem # @schema type: string; enum: []; description: Defines if a volume is intended to be used with a formatted filesystem or to remain in raw block state
```

Comment:
`enum: []`
Do we allow any value here?

Reply:
According to tests: yes! But anyway, I've added `filesystem, block` now, I've just forgotten to add this in the first place, thanks for noticing